app/controllers/admin/worldwide_offices_controller.rb

```diff
@@ -71,6 +71,7 @@ def worldwide_office_params
                     :id, :title, :contact_type_id, :comments, :recipient,
                     :street_address, :locality, :region, :postal_code,
                     :country_id, :email, :contact_form_url,
-                    contact_numbers_attributes: [:id, :label, :number, :_destroy]])
+                    contact_numbers_attributes: %i[id label number _destroy]
+])
```

Comment:
It would be better to break up and indent this `])` so the `]` lines up with the contact_attributes: `[` that opened it, and the `)` lines up with the `.permit(` line that opens it.